### DISCUSSION TITLE
Episode 173: Parallel chunk_by

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#65] [Episode]
Episode Link: https://adspthepodcast.com/2024/03/15/Episode-173.html

Feel free to write comments or ask questions 🙂

**Comment by RishabhRD** [comment]
> Scan is not an algorithm, scan is a communication mechanism. Scan and non commutative reduction where information goes from left to right are communication mechanism. They are synchronization primitives for threads to communicate to threads that are later in computation.


Which episode have a detailed discussion on this line?

**Reply by codereport** [reply]
* https://adspthepodcast.com/2021/05/14/Episode-25.html
* https://adspthepodcast.com/2021/08/06/Episode-37.html

**Comment by waydan** [comment]
Did Connor and Bryce get Freaky Fridayed?
